mm2src/coins/solana.rs

```diff
@@ -374,6 +373,19 @@ impl MarketCoinOps for SolanaCoin {
         Box::new(fut.boxed().compat())
     }
 
+    fn send_raw_tx_bytes(&self, tx: &[u8]) -> Box<dyn Future<Item = String, Error = String> + Send> {
+        let coin = self.clone();
+        let tx = tx.to_owned();
+        let fut = async move {
+            let tx: Transaction = deserialize(tx.as_slice())
+                .map_to_mm(|e| e)
+                .map_err(|e| format!("{:?}", e))?;
+            let signature = coin.rpc().send_transaction(&tx).await.map_err(|e| format!("{:?}", e))?;
```

Comment:
It's not a signature :slightly_smiling_face:

Reply:
Hmm, `send_transaction` returns `solana_sdk::signature::Signature`. Am I missing something ? :)

Reply:
https://docs.solana.com/ru/terminology#transaction-id - as Solana uses first signature of the transaction as ID, and we require returning transaction hash/ID, it's fine. Please mark it as resolved once you read it 🙂